cmd/tracee-rules/output.go

```diff
@@ -39,10 +41,16 @@ func setupTemplate(inputTemplateFile string) (*template.Template, error) {
 	}
 }
 
-func setupOutput(w io.Writer, webhook string, webhookTemplate string, contentType string, outputTemplate string) (chan detect.Finding, error) {
+func setupOutput(w io.Writer, fluentForwardUrl string, fluentForwardTemplate string, fluentForwardTag string, webhook string, webhookTemplate string, contentType string, outputTemplate string) (chan detect.Finding, error) {
 	out := make(chan detect.Finding)
 	var err error
 
```

Comment:
if fluentForwardTemplate != "" {
	var tFluentForward *template.Template
	tFluentForward, err = setupTemplate(fluentForwardTemplate)
	if err != nil  {
		return nil, fmt.Errorf("error preparing Fluent Forward template: %v", err)
	}
}

Reply:
I was following the original approach in the code so will leave this for now.
It would also mean the tFluentForward variable is not in scope for use later.